docs/extensions/blender_addon.md

```diff
@@ -44,12 +44,12 @@ This Blender add-on allows for compositing with a Nerfstudio render as a backgro
 
    - In object properties, select "Shadow Catcher". This makes the representation invisible in the render, but all shadows cast on it will render. You may have to switch to the cycles renderer to see the shadow catcher option.
    <center>
-   <img width="300" alt="image" src="https://user-images.githubusercontent.com/9502341/211244787-859ca9b5-6ba2-4056-aaf2-c89fc6370c2a.png">
+   <img width="200" alt="image" src="https://user-images.githubusercontent.com/9502341/211244787-859ca9b5-6ba2-4056-aaf2-c89fc6370c2a.png">
```

Comment:
Any reason to have these images smaller? I feel like they are difficult to read 
<img width="932" alt="image" src="https://user-images.githubusercontent.com/3310961/232280861-bdc76407-ae23-47c8-9637-3af70f9fb357.png">

Reply:
Fixed in new commit